sw/airborne/modules/dcf/dcf.h

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (C) 2017  Hector Garcia de Marina
+ *
+ * This file is part of paparazzi.
+ *
+ * paparazzi is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2, or (at your option)
+ * any later version.
+ *
+ * paparazzi is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with paparazzi; see the file COPYING.  If not, write to
+ * the Free Software Foundation, 59 Temple Place - Suite 330,
+ * Boston, MA 02111-1307, USA.
+ *
+ */
+
+/** @file dcf.h
+ *
+ *  Distributed circular formation algorithm
+ */
+
+#ifndef DCF_H
+#define DCF_H
+
+#include "std.h"
+
+/*! Default number of neighbors per aircraft */
+#ifndef DCF_MAX_NEIGHBORS
+#define DCF_MAX_NEIGHBORS 4
+#endif
+
+struct dcf_con {
+  float k;
+  float radius;
+  uint16_t timeout;
+  float theta;
+  uint16_t broadtime;
+};
+
+extern struct dcf_con dcf_control;
+
+struct dcf_tab {
+  int16_t tableNei[DCF_MAX_NEIGHBORS][4];
+  int16_t error_sigma[DCF_MAX_NEIGHBORS];
+  uint32_t last_theta[DCF_MAX_NEIGHBORS];
+};
+
+extern struct dcf_tab dcf_tables;
```

Comment:
wouldn't it be cleaner (or more complicated) to have a table of structure rather than a structure of tables ?